domains/asteriamp.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "optixyt0",
+        "email": "kaibeeley@icloud.com"
+    },
+    "record": {
+        "A": ["185.27.134.179"]
```

Comment:
cant access site "This page isn’t working. 185.27.134.179 didn’t send any data."

Reply:
should I just send my url?

Reply:
you can

Reply:
asteriamp.rf.gd

Reply:
@CuteDog5695